prisma/migrations/20220829024459_rename_rejected_items_and_make_topic_optional/migration.sql

```diff
@@ -0,0 +1,27 @@
+/*
+  Warnings:
+
+  - You are about to drop the `RejectedCuratedCorpusItem` table. If the table is not empty, all the data it contains will be lost.
+
+*/
+-- DropTable
+DROP TABLE `RejectedCuratedCorpusItem`;
+
```

Comment:
Actually, I'm going to close this one. And redo it again without touching the name of the table, because dropping the table will not do!